<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ground Truth System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="m_centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_workbenchMenu">
    <property name="title">
     <string>&amp;Workbench</string>
    </property>
    <addaction name="m_newWorkbenchAction"/>
    <addaction name="m_openWorkbenchAction"/>
    <addaction name="m_saveWorkbenchAction"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="m_toolMenu">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_helpAction"/>
    <addaction name="separator"/>
    <addaction name="m_aboutAction"/>
    <addaction name="m_aboutQtAction"/>
   </widget>
   <addaction name="m_workbenchMenu"/>
   <addaction name="m_toolMenu"/>
   <addaction name="m_helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="m_statusbar"/>
  <action name="m_newWorkbenchAction">
   <property name="text">
    <string>&amp;New Workbench...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_openWorkbenchAction">
   <property name="text">
    <string>&amp;Open Workbench...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="m_saveWorkbenchAction">
   <property name="text">
    <string>&amp;Save Workbench</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_helpAction">
   <property name="text">
    <string>&amp;Help Contents</string>
   </property>
  </action>
  <action name="m_aboutAction">
   <property name="text">
    <string>&amp;About GTS</string>
   </property>
  </action>
  <action name="m_aboutQtAction">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
